### ISSUE TITLE
How to get object label per class instance?

### ISSUE THREAD

**Issue by twarrendewit** [Unity-Technologies/com.unity.perception#407]
The InstanceSegmentationLabeler gives me good ground truth on individual object with the following JSON in captures:

```
{
  "instance_id": 2,
  "color": {
    "r": 0,
    "g": 74,
    "b": 255,
    "a": 255
  }
```

However, the SemanticSegmentationLabeler gives me the object class according to the color key in annotation_definitions.json.

What I would like is to combine the two: augment my JSON output with class label but leave the color map alone, eg, "object with color [RGBA] is of type [CAR/BOAT]" according to the IdLabelConfig.

How would I go about augmenting the Instance Segmentation data with the Semantic Segmentation label set? Am I missing a switch or will I need to combine these two entire classes?

**Comment by twarrendewit** [comment]
Upon further review, I seem to be able to get the two out of using RenderedObjectInfo labeler and looking in the label_id field of each metrics entry to tie back to the labels in metric_definitions.json (mirrored by annotation_definitions.json). Is this the recommended methodology?

**Comment by JonathanHUnity** [comment]
Thanks for the detailed question @twarrendewit . This would be useful information to include in the Instance Segmentation output. I'll leave this issue open as an enhancement. For now, using the RenderedObjectInfo is a fine approach as a workaround.

**Comment by an99990** [comment]
where can i find RenderedObjectInfo label @twarrendewit ?
unity 2020.3.32f1
unity perception 0.10.0

thank you

**Comment by JonathanHUnity** [comment]
@an99990 The RenderedObjectInfoLabeler can be found under Labelers on the PerceptionCamera.

**Comment by StevenBorkman** [comment]
Hey all! The Perception team just released 🎉 Perception 1.0 🎉, a major update to the toolset! With this release we have introduced a new output format SOLO, a more efficient way to store and read-in data. With SOLO we have added the ability to write out the label name with the segmentation results.

There are a bunch of cool features to check out such as ray tracing, depth, a new output format SOLO, normals, and more! Here is the full list of changes: [Perception 1.0 Changelog](https://github.com/Unity-Technologies/com.unity.perception/blob/main/com.unity.perception/CHANGELOG.md)
